L 07/02/2020 - 23:59:21: Log file started (file "logs\L0702037.log") (game "D:\sup\steamapps\common\GarrysModDS\garrysmod") (version "7836")
L 07/02/2020 - 23:59:21: server_cvar: "sv_alltalk" "3"
L 07/02/2020 - 23:59:22: Connection to Steam servers successful.
L 07/02/2020 - 23:59:22:    Public IP is 31.130.42.26.
L 07/02/2020 - 23:59:22: Assigned anonymous gameserver Steam ID [A-1:3870896128(15079)].
L 07/02/2020 - 23:59:22: VAC secure mode is activated.
L 07/03/2020 - 00:15:09: Lua Error: 
[ERROR] addons/badmin/lua/ba/core/util/svars/svars_sv.lua:34: attempt to call method 'query_sync' (a nil value)
  1. loadSvars - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:34
   2. unknown - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:41


L 07/03/2020 - 00:15:09: Lua Error: 
[ERROR] addons/badmin/lua/ba/core/util/svars/svars_sv.lua:34: attempt to call method 'query_sync' (a nil value)
  1. loadSvars - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:34
   2. unknown - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:41


L 07/03/2020 - 00:15:32: Lua Error: 
[ERROR] addons/badmin/lua/ba/core/util/svars/svars_sv.lua:36: bad argument #1 to 'pairs' (table expected, got nil)
  1. pairs - [C]:-1
   2. loadSvars - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:36
    3. unknown - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:41


L 07/03/2020 - 00:15:36: Lua Error: 
[ERROR] addons/badmin/lua/ba/core/util/svars/svars_sv.lua:36: bad argument #1 to 'pairs' (table expected, got nil)
  1. pairs - [C]:-1
   2. loadSvars - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:36
    3. unknown - addons/badmin/lua/ba/core/util/svars/svars_sv.lua:41


L 07/03/2020 - 00:15:43: server_message: "quit"
L 07/03/2020 - 00:15:43: Log file closed.
